### PR TITLE
feat: initial revision for e2e tests

### PR DESCRIPTION
**Pull request by barbmarcio**
This PR adds End-to-end tests to the SDK.

Closes: https://topsort.atlassian.net/browse/API-922

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
+.env
+
 coverage/
 dist/
 node_modules/
+test-results/
```
Comment (jbergstroem) [inline .gitignore:6]: do we need this?

Comment (barbmarcio) [reply .gitignore:6]: i've added to not push the test-results to the branches, just like is usually done for coverage reports
is it better to have them?

**Review by jbergstroem — APPROVED** [review]

**Comment by jbergstroem** [comment]
Ok to land but lets do the other PR first.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -94,6 +95,16 @@ To run the unit tests, use the following command:
...
 ```
 
+### E2E Tests
+
+By default, the application is set to serve a web browser in the port `8080` for Playwright. If this port is already being used on your local machine, make sure you change the port in your `.env`.`PLAYWRIGHT_PORT`.
+To run the end-to-end tests, make sure you have latest bundled file before running it. Use the following commands:
+
+```bash
+bun run build
+bun run test:e2e
```
Comment (jbergstroem) [inline CONTRIBUTING.md:105]: don't you have to download the browsers before running this?

Comment (barbmarcio) [reply CONTRIBUTING.md:105]: Yes we do! let me update
Thanks for catching that

**Pull request merged**